src/patternfly/layouts/Toolbar/styles.scss

```diff
@@ -26,20 +33,25 @@
 // Section
 .pf-l-toolbar__section {
   flex-basis: 100%;
-
-  // Add margin-top for stacked sections
-  &:not(:first-child) {
-    margin-top: var(--pf-l-toolbar__section--MarginTop);
-  }
 }
 
 // Group
-.pf-l-toolbar__group:not(:last-child) {
-  margin-right: var(--pf-l-toolbar__group--MarginRight);
+.pf-l-toolbar__group {
+  padding: var(--pf-global--spacer--xl);
+
+  // Add margin-top for stacked groups
+  margin-top: var(--pf-l-toolbar__section--MarginTop);
+
+  // Set margin-right so groups will align when wrapped
+  .pf-l-toolbar__group:not(:last-child) {
+    margin-right: var(--pf-l-toolbar__group--MarginRight);
+  }
 }
 
 // Item
 .pf-l-toolbar__item {
+  margin: 0;
```

Comment:
This doesn't do anything since there is no `margin` on toolbar items currently. Also, you generally want to use a variable instead of a number for spacing values like this.

Reply:
Code has been removed from latest build.